src/index.html

```diff
@@ -12,15 +12,33 @@
     <script nomodule src="/build/shadowcat.js"></script>
   </head>
   <body>
+    <!-- Our typical oauth workflow using puma includes several server-side
+      redirects in order to get the auth token. This component tests a similar
+      workflow using the mock server. -->
     <manifold-oauth
-      id="oauth"
-      oauth-url="https://shadowcat-test-server.herokuapp.com/signin/oauth/web"
+      id="server-side"
+      oauth-url="https://manifold-shadowcat-test-server.herokuapp.com/signin/oauth/web"
```

Comment:
Should we make this configurable? In theory, it'll live under `https://login.manifold.co/signin/oauth/web`, but I can see the point in allowing us to change it easily for tests etc.

(not sure if that's easy to do or not)

Reply:
These are just the dev server examples, so we can change this to whatever we want when we use it elsewhere.

Reply:
Oh right! For some reason I thought this was going to be embedded on the platform's page, but that's not the case.

Follow up question: do we have to provide the oauth-url? Could we make it easier on the platforms and just provide a default (and allow for overwriting here)?

Reply:
![image](https://user-images.githubusercontent.com/10498708/60969134-22708f00-a2f5-11e9-97f9-099e5409e8fe.png)

Yup! So the default should be `https://login.manifold.co/signin/oauth/web`, correct?

Reply:
We'll handle that in another PR

Reply:
👌 and yep, that'd be the one!

Reply:
Would providing the default be better here, or in the wrapper component in `ui`?

Reply:
i think here. it's only going to be used once per platform so if the platform handled it I think it would end up being the same amount of code maybe? idk, if that's a strong justification.